build.gradle

```diff
@@ -13,7 +13,7 @@ buildscript {
 apply plugin: 'net.minecraftforge.gradle.forge'
 
 
-version = "1.10.2-1.3.11" 
+version = "1.10.2-1.3.12"
```

Comment:
I'm not sure what azanors stand on this is, but imo external pull requests shouldnt change the project version number. Thats the job of the maintainer and should only be done if there is a new build.